rack-test.gemspec

```diff
@@ -1,25 +1,26 @@
 # -*- encoding: utf-8 -*-
+# stub: rack-test 0.6.3 ruby lib
```

Comment:
I ran `thor` command on Ruby 2.2.7 when I generated the gemspec file. The Ruby's RubyGems was used to append the line. Maybe.
Because when I tried it on Ruby 2.4, `freeze` string was appended. and there was so much difference between before and after modification. I did not like the difference.